lib/parser.js

```diff
@@ -195,7 +210,7 @@ Parser.prototype._parse = function (element, context) {
       element.attribs.isOpen = hasIsOpen ? hasIsOpen : false;
       try {
         if (fs.statSync(element.attribs.src).isFile()) {
-          let resultDir = path.dirname(path.join('{{hostBaseUrl}}', path.relative(process.cwd(), element.attribs.src)));
+          let resultDir = path.dirname(path.relative('{{hostBaseUrl}}', element.attribs.src));
```

Comment:
Why is this change necessary?

Reply:
This will break when hosted on GitHub:

<img width="900" alt="screenshot 2018-01-27 02 16 11" src="https://user-images.githubusercontent.com/14091939/35454224-17e190b8-0308-11e8-9e73-0f5a5168939b.png">

Reply:
If a dynamic panel includes a boilerplate file,
I will replace the `filePath` and `element.attribs.src` with the boilerplate filepath. However, it does not work correctly as it is unable to retrieve the content from source: `/\_boilerplates\filename._include_.html`.
 
I tried normalizing the filepath and replacing it with the above line. It works locally during testing but I didn't know it will break when hosted on Github.

Reply:
The `{{baseUrl}}` is `/website` → https://nus-cs2103-ay1718s2.github.io/website/

Doing `../schedule` will give https://nus-cs2103-ay1718s2.github.io/schedule/
It's important that rendered files do not contain relative paths for `include` to work well.